.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+go:
+  - 1.6.x
+  - 1.7.x
+  - 1.8.x
+  - tip
```

Comment:
FWIW, Travis refers to `tip` as `master` now. See https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use. `tip` still works, but it's unclear if it'll stop working in the future.